src/service/apiClient.js

```diff
@@ -52,4 +52,4 @@ async function request(method, endpoint, data, auth = true) {
   return response.json();
 }
 
-export { login, getPosts, register, createProfile };
+export { login, getPosts, register, createProfile, get };
```

Comment:
don't expose `get` -> create instead a getUsers function instead in `apiClient.js` and export that function.